scripts/protocgen.sh

```diff
@@ -1,29 +1,37 @@
 #!/usr/bin/env bash
-
-set -eo pipefail
+set -euo pipefail
 
 : ${VERS:=master}
-URL_PATH=archive/
+URL_PATH=tarball/
 if [[ VERS -ne master ]]; then
-    URL_PATH=archive/refs/tags/v
+    URL_PATH=tarball/refs/tags/v
 fi
 
+echo "fetching proto files"
+
+# Get shortened ref of commit
+REF=`curl -H "Accept: application/vnd.github.v3.sha" -qL "https://api.github.com/repos/tendermint/spec/commits/${VERS}" | cut -c -7`
 # Edit this line to clone your branch, if you are modifying protobuf files
-curl -qL "https://github.com/tendermint/spec/${URL_PATH}${VERS}.tar.gz" | tar -xjf - spec-"$VERS"/proto/
+curl -qL "https://api.github.com/repos/tendermint/spec/${URL_PATH}${REF}" | tar -xzf - tendermint-spec-"$REF"/
+
+cp -r ./tendermint-spec-"$REF"/proto/tendermint/** ./proto/tendermint
```

Comment:
I don't think `**` does what you maybe intended here. As far as I know this is just "anything, twice", not a recursive glob.

In this case it doesn't matter since we're using `-r` anyway, though.